helpers.py

```diff
@@ -34,3 +34,12 @@ def setup_logger():
     console.setFormatter(logging.Formatter(log_format, datefmt=log_datefmt))
     logging.getLogger('').addHandler(console)
     logging.getLogger('urllib3').setLevel(logging.WARNING)
+
+
+def sign_message_with_static_key(message):
```

Comment:
This method is used only by tests - maybe it's worth having separate `test/helpers.py` for test helpers?

Reply:
moved to tests/utils.py